src/fastmcp/server/http.py

```
@@ -242,8 +242,14 @@ async def sse_endpoint(request: Request) -> Response:
 
     @asynccontextmanager
     async def lifespan(app: Starlette) -> AsyncGenerator[None, None]:
```

Ruff (0.14.10)

Unused function argument: `app`

(ARG001)